content/v2.0/reference/syntax/annotated-csv.md

```diff
@@ -177,29 +180,81 @@ csvData = "#datatype,string,long,dateTime:RFC3339,dateTime:RFC3339,dateTime:RFC3
 #group,false,false,false,false,false,false,false,false
 #default,,,,,,,,
 ,result,table,_start,_stop,_time,region,host,_value
-,my-result,0,2018-05-08T20:50:00Z,2018-05-08T20:51:00Z,2018-05-08T20:50:00Z,east,A,15.43
-,my-result,0,2018-05-08T20:50:00Z,2018-05-08T20:51:00Z,2018-05-08T20:50:20Z,east,B,59.25
-,my-result,0,2018-05-08T20:50:00Z,2018-05-08T20:51:00Z,2018-05-08T20:50:40Z,east,C,52.62
-,my-result,1,2018-05-08T20:50:00Z,2018-05-08T20:51:00Z,2018-05-08T20:50:00Z,west,A,62.73
-,my-result,1,2018-05-08T20:50:00Z,2018-05-08T20:51:00Z,2018-05-08T20:50:20Z,west,B,12.83
-,my-result,1,2018-05-08T20:50:00Z,2018-05-08T20:51:00Z,2018-05-08T20:50:40Z,west,C,51.62
+,,0,2018-05-08T20:50:00Z,2018-05-08T20:51:00Z,2018-05-08T20:50:00Z,east,A,15.43
+,,0,2018-05-08T20:50:00Z,2018-05-08T20:51:00Z,2018-05-08T20:50:20Z,east,B,59.25
+,,0,2018-05-08T20:50:00Z,2018-05-08T20:51:00Z,2018-05-08T20:50:40Z,east,C,52.62
+,,1,2018-05-08T20:50:00Z,2018-05-08T20:51:00Z,2018-05-08T20:50:00Z,west,A,62.73
+,,1,2018-05-08T20:50:00Z,2018-05-08T20:51:00Z,2018-05-08T20:50:20Z,west,B,12.83
+,,1,2018-05-08T20:50:00Z,2018-05-08T20:51:00Z,2018-05-08T20:50:40Z,west,C,51.62
 "
 
 csv.from(csv: csvData)
 ```
 
-### Valid data types
+### Annotated CSV with linepart annotations
+The `linepart` annotation tells the [`influx write` command](/v2.0/reference/cli/influx/write/)
+how to parse the annotated CSV into line protocol when writing to InfluxDB.
+
+```
+#linepart measurement,tag,tag,field,field,ignored,time
+m,cpu,host,time_steal,usage_user,nothing,time
+cpu,cpu1,host1,0,2.7,a,1482669077000000000
+cpu,cpu1,host2,0,2.2,b,1482669087000000000
+```
+
+#### Line protocol and linepart annotations
+| Linepart key         | Description                                                           |
+|:------------         |:-----------                                                           |
+| `measurement`        | column value parsed as the measurement                                |
```

Comment:
The implementation doesn't parse column value as a measurement, one might think we expect measurement as it appears in the line protocol. I am not a native English speaker, but it would not be confusing to me if there would be just: column value is the measurement. Applies to all descriptions in this table.